### PR TITLE
FAQ updates

### PR DESCRIPTION
**Pull request by matthchr**
**If applicable**:
- [x] this PR contains documentation
- [ ] this PR contains tests

### CODE REVIEW

File: docs/hugo/content/introduction/frequently-asked-questions.md [hunk 1]
```diff
@@ -3,6 +3,19 @@ title: FAQ
...
+
+We ship updates to ASO as needed, with an eye towards releasing every 1-2 months. If there are urgent fixes, a release may happen
+more quickly than that. If there haven't been any major changes (or there are ongoing major changes that are taking a long time) a 
+release may happen more slowly.
+
+### What is the support model?
+
+Azure Service Operator is an officially supported Microsoft OSS product. Currently, support is done via GitHub issues or
+the `azure-service-operator` channel of the [Kubernetes Slack](https://kubernetes.slack.com/). There are plans to integrate ASO into
+AKS as an addon after ASO has officially gone GA. At that point, support for ASO as an AKS addon would be achieved by raising an Azure
```
Comment (theunrepentantgeek) [inline docs/hugo/content/introduction/frequently-asked-questions.md:16]:
```suggestion
AKS as an addon after ASO has officially gone GA. At that point, support for ASO as an AKS addon would be accessed by raising an Azure 
```

**Review by theunrepentantgeek — APPROVED** [review]
One suggested wording tweak. LGTM.

**Review by super-harsh — APPROVED** [review]

**Pull request merged**